phac-nml/irida-next
# Clipboard component for copying PUID's

## What does this PR do and why?
_Describe in detail what your merge request does and why._

I created a clipboard view component with stimulusjs controller that allows copying to the clipboard of a specified value with the click of a button.  This will help standardize this across all views.

This was added to:

1. Page Header component on the attribute id
2. Groups dashboard page, every group puid can be copied

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

![image](https://github.com/user-attachments/assets/24abf085-6a12-4a47-8f7b-68351709fd3a)

![image](https://github.com/user-attachments/assets/878ebf45-5067-42b1-9c4c-2ea411ca8edb)

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. Go to the groups dashboard page
2. On any row click on the copy to clipboard button
3. Assert that the `Copied` tooltip was displayed
4. Try to paste the value somewhere, it should be the puid from the row.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
